inc/utilities.php

```diff
@@ -287,3 +287,53 @@ function m_getlang() {
 	return reset( $langs );
 
 }
+
+/**
+ *
+ * @param string	$folder
+ * @param string	$file
+ * @param array	$args
+ * 
+ * @return void|false
+ */
+
+function m_get_template($folder, $file, $args = array()) {
+	get_template_part('template-parts/'.$folder.'/'.$file, '', $args);
```

Comment:
ti propongo 
`
function m_part( $folder, $file, $args = array() ) {
	get_template_part( 'template-parts/' . $folder, $file, $args );
}`

che é piu corto :) 
il gioco dei parts mi sa no ti é chiaro
in pratica se tu fait
get_template_part('template-parts/card-post','big'):
lui cerca card-post-big . se non la trova usa card-post generica
cosa comodissima da sfruttare. guarda la pr di mosne card :)

Reply:
La cosa dei parts la sapevo, ma nella pratica non la uso mai, per questo ho saltato lo slug nella funzione.
Mi piace che sia m_part più corto, posso pure aggiungere l'argomento slug, alla fine a me mi pesa il culo scrivere sempre tutto il percorso dei template.